### PR TITLE
Update Greeting Message

### LINKED ISSUE #1: Modify Greeting "Hello World"

### PR DESCRIPTION
**Pull request by dlaylay**
This pull request updates the greeting message from 'Hello World' to 'Hello Deanne Laylay' and includes a .gitignore file.

### CODE REVIEW

File: MyCoreApp/Pages/Index.cshtml [hunk 1]
```diff
@@ -5,6 +5,6 @@
 }
 
 <div class="text-center">
-    <h1 class="display-4">Hello World!</h1>
+    <h1 class="display-4">Hello Deanne Laylay!</h1>
```
Comment (dila-liyanage) [inline MyCoreApp/Pages/Index.cshtml]: Please remove exclamation mark here.

**Review by dila-liyanage — CHANGES_REQUESTED** [review]
Added minor feedback in index.cshtml file. Please check

File: MyCoreApp/Pages/Index.cshtml (hunk 1 above)
Comment (dlaylay) [reply MyCoreApp/Pages/Index.cshtml]: Done

**Review by dila-liyanage — APPROVED** [review]
LGTM!

**Pull request merged**